port/blueprint/readStateToPortBody.go

```diff
@@ -131,3 +132,84 @@ func calculationPropertiesToBody(ctx context.Context, state *BlueprintModel) map
 
 	return calculationProperties
 }
+
+func aggregationPropertiesToBody(ctx context.Context, state *BlueprintModel) (map[string]cli.BlueprintAggregationProperty, error) {
+	aggregationProperties := map[string]cli.BlueprintAggregationProperty{}
+
+	for identifier, prop := range state.AggregationProperties {
+		aggregationProp := cli.BlueprintAggregationProperty{
+			Target: prop.Target.ValueString(),
+		}
+
+		if !prop.Title.IsNull() {
+			title := prop.Title.ValueString()
+			aggregationProp.Title = &title
```

Comment:
can't you simply do this?
```suggestion
			aggregationProp.Title = prop.Title.ValueString()
```